examples/kprobe_percpu/main.go

```diff
@@ -0,0 +1,81 @@
+//go:build linux
+// +build linux
+
+// This program demonstrates attaching an eBPF program to a kernel symbol.
+// The eBPF program will be attached to the start of the sys_execve
```

Comment:
Mention the use of a per CPU map instead?

Reply:
😅 I overlooked these lines.